src/Compilers/VisualBasic/Portable/VBResources.resx

```diff
@@ -1039,6 +1042,9 @@
   <data name="ERR_InvalidOverrideDueToReturn2" xml:space="preserve">
     <value>'{0}' cannot override '{1}' because they differ by their return types.</value>
   </data>
+  <data name="ERR_InvalidOverrideDueToTupleNames" xml:space="preserve">
+    <value>'{0}' cannot override '{1}' because they differ by their tuple element names.</value>
```

Comment:
Hum. That could read wrong and imply that if they differ by tuple names and something else then it would override.